src/index.jsx

```diff
@@ -2,7 +2,64 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
 
+export const numbers1 = {
+  a: 2,
+  b: 3,
+};
+
+export const numbers2 = {
+  a: -5,
+  b: 5,
+};
+
+export const numbers3 = {
+  a: 10,
+  b: 0,
+};
+
+export const numbers4 = {
+  a: 0,
+  b: 5,
+};
+
+export const numbers5 = {
+  a: 0,
+  b: 0,
+};
```

Comment:
- you shouldn't declare objects with numbers. Just pass them right away as arguments